spec/controllers/v1/legislators_controller_spec.rb

```diff
@@ -22,38 +19,60 @@
     end
 
     context "good email" do
-      context "new user" do
-        before { get :index, { email: 'user@example.com' } }
+      let!(:campaign) { FactoryGirl.create(:campaign_with_reps, count: 3, priority: 1) }
+      let (:state)    { FactoryGirl.create(:state, abbrev: 'CA') }
+      let (:district) { FactoryGirl.create(:district, district: '13', state: state) }
+      let!(:senator)  { FactoryGirl.create(:senator, state: state, with_us: false) }
+      let!(:rep)      { FactoryGirl.create(:representative, district: district,
+                                                            with_us: true) }
+      let(:campaign_ids) { campaign.legislators.pluck(:id) }
+      let(:returned_ids) { assigns(:target_legislators).map{|tl| tl['id']} }
```

Comment:
Nice one!

I feel like a lot of these tests would be a lot simpler with fixtures -- setting up a campaign for each test is a nightmare. Is it possible to use fixtures and factories in the same project?